secant/UI Components/Chips/EnumeratedChip.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  EnumeratedChip.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 10/19/21.
+//
+
+import SwiftUI
+
+struct EnumeratedChip: View {
+    @Clamping(1...24)
+    var index: UInt = 1
+
+    var text: String
+
+    var body: some View {
+        NumberedText(
+            number: index,
+            text: text
+        )
+        .frame(
+            minWidth: 0,
+            maxWidth: .infinity,
+            minHeight: 30,
+            maxHeight: .infinity,
+            alignment: .leading
+        )
+        .padding(.horizontal, 16)
+        .padding(.vertical, 8)
+
+        .background(
+            Asset.Colors.Buttons.primaryButtonPressed.color
+        )
+        .cornerRadius(6)
+    }
+}
+
+struct NumberedText: View {
+    @Clamping(1...24) var number: UInt = 1
```

Comment:
This value is already clamped elsewhere, duplicating the logic here will make it harder to update in the future.  If someone were to update the Chip view for example to clamp 1...30 and it didn't work, it would be confusing why.  Centalizing this logic to a single location is always best.  Here you can just use a regular old Int since this view doesn't care about the validation, it's the Chip view that wants to bring this Text together that cares about which values are used.